src/view.js

```diff
@@ -272,6 +268,10 @@ function bindEvents(ctrl, el, context) {
 }
 
 function renderBoard(ctrl) {
+  ctrl.pointerSelected = function pointerSelected() {
+    return !ctrl.sparePieceSelected || ctrl.sparePieceSelected === 'pointer';
+  };
```

Comment:
I don't understand why this function is declared here. Why not in ctrl.js instead? Sounds quite natural to me.